documentation/src/main/asciidoc/topics/proc_server_creating_patches.adoc

```diff
@@ -0,0 +1,85 @@
+[id='patching_create-{context}']
+= Creating Server Patches
+You can create patches for {brandname} servers from an existing server
+installation.
+
+ifndef::productized[]
+You can create patches for {brandname} servers starting from 10.1.7. You can
+patch any 10.1 or later server installation. However you cannot patch 9.4.x or
+earlier servers with 10.1.7 or later.
+
+You can also create patches that either upgrade or downgrade the {brandname}
+server version. For example, you can create a patch from version 10.1.7 and use
+it to upgrade version 10.1.5 or downgrade version 11.0.0.
+endif::productized[]
+ifdef::productized[]
+You can create patches for {brandname} servers starting from 8.0.1. You can
+patch 8.0 GA servers with 8.0.1. However you cannot patch 7.3.x or earlier
+servers with 8.0.1 or later.
+
+You can also create patches that either upgrade or downgrade the {brandname}
+server version. For example, you can create a patch from version 8.0.1 and use
+it to upgrade version 8.0 GA or downgrade a later version.
+
+[IMPORTANT]
+====
+To avoid issues with server installations, {redhat} recommends that you
+download server patches from the {portal} rather than creating them.
+====
+endif::productized[]
+
+.Procedure
+
+. Navigate to `{server_home}` for a {brandname} server installation that has
+the target version for the patch you want to create.
+. Start the CLI.
++
+[source,options="nowrap",subs=attributes+]
+----
+$ bin/cli.sh
+[disconnected]>
+----
++
+. Use the `patch create` command to generate a patch archive and include the `-q` option with a meaningful qualifier to describe the patch.
++
+[source,options="nowrap",subs=attributes+]
+----
+[disconnected]> patch create -q "this is my test patch" path/to/mypatch.zip . path/to/server/home
```

Comment:
suggestion: don't use `.` to signal the server path. It may be confusing IMO.
use something similar `path/to/target/server/home` and `path/to/source/server/home`

Reply:
thanks, Pedro. I've updated with this.